### PR TITLE
Added SVG sprite task.

### PR DESCRIPTION
**Pull request by DamianMullins**

### CODE REVIEW

File: tasks/images.js [hunk 1]
```diff
@@ -45,3 +57,43 @@ gulp.task('images:optimise', () => gulp.src(`${pathBuilder.imgSrcDir}/**`)
...
+
+/**
+ * images:svg-sprite Task
+ * -------------
+ * Generate an SVG sprite using svgstore.
+ *
+ */
+gulp.task('images:svg-sprite', () => gulp.src(`${pathBuilder.imgSrcDir}/**/*.svg`, {
+    base: pathBuilder.imgSrcDir
+})
```
Comment (ashleynolan) [inline tasks/images.js]: This line should be indented so it’s on the same level as the pipe statements

Comment (DamianMullins) [inline tasks/images.js]: Fixed.

File: config.js [hunk 2]
```diff
@@ -43,7 +43,8 @@ const ConfigOptions = () => {
          * Image-related variables
          */
         img: {
-            imgDir: 'img'
+            imgDir: 'img',
+            svgSpriteFilename: 'sprite.svg'
```
Comment (zlatinivanov-je) [inline config.js]: Should that be `svgSpriteFileName` instead

**Comment by zlatinivanov-je** [comment]
👍

File: config.js (hunk 2 above)
Comment (DamianMullins) [inline config.js]: I did wonder the same, then [came across this](https://english.stackexchange.com/questions/5366/which-is-correct-filename-file-name-or-filename). It's on the internet so it must be correct, right? 😜

Comment (zlatinivanov-je) [inline config.js]: This was just a suggestion. I don't mine whichever way. 😉

**Pull request merged**